models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -367,30 +367,44 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     if ("GDD" %in% pft.names) {
       param[which(param[, 1] == "gddLeafOn"), 2] <- pft.traits[which(pft.names == "GDD")]
     }
-
+    
     # Fraction of leaf fall per year (should be 1 for decid)
     if ("fracLeafFall" %in% pft.names) {
       param[which(param[, 1] == "fracLeafFall"), 2] <- pft.traits[which(pft.names == "fracLeafFall")]
     }
-
+    
     # Leaf growth.  Amount of C added to the leaf during the greenup period
     if ("leafGrowth" %in% pft.names) {
       param[which(param[, 1] == "leafGrowth"), 2] <- pft.traits[which(pft.names == "leafGrowth")]
     }
   }  ## end loop over PFTS
   ####### end parameter update
-
-
+  
+  
   #### write INITIAL CONDITIONS here ####
   if (!is.null(IC)) {
     ic.names <- names(IC)
     ## plantWoodInit gC/m2
     plant_wood_vars <- c("AbvGrndWood", "abvGrndWoodFrac", "coarseRootFrac", "fineRootFrac")
     if (all(plant_wood_vars %in% ic.names)) {
-  
-      
       # reconstruct total wood C
       wood_total_C <- IC$AbvGrndWood / IC$abvGrndWoodFrac
+      #Sanity check
+      if (is.na(wood_total_C) | is.infinite(wood_total_C) | is.nan(wood_total_C) | wood_total_C <0) {
+        wood_total_C <- 0
```

Comment:
@para2x @bailsofhay what is this check for? Is this supposed to be 0 or should it be something like 0.001?

Reply:
Dunno, ask @para2x

Reply:
@serbinsh I have the complete version of this check in my PR where it gives warnings and etc that what happed. It's for when the woodFrac comes out as zero and blows up the wood_total.